dashboard/src/modules/components/SidebarComponent/sideMenuOptions.js

```diff
@@ -30,6 +30,13 @@ export const menuOptions = [
         submenuOf: "dashboard",
         link: "results",
       },
+      {
+        name: "Comparison",
+        submenu: true,
+        key: "comparsion",
+        submenuOf: "dashboard",
+        link: "comparsion",
+      },
```

Comment:
Here and in `routeConstants.js` "comparison" is mis-spelled as "comparsion".

Reply:
:grinning: